cep-recipe-serialization.md

```diff
@@ -0,0 +1,1141 @@
+        build_number: 0
+        depends: []
+        license: LGPL-2.1 and GPL-2.0
+        md5: 39c6b54e94014701dd157f4f576ed211
+        name: xz
+        sha256: 59d78af0c3e071021cfe82dc40134c19dab8cdf804324b62940f5c8cd71803ec
+        size: 235693
+        subdir: osx-arm64
+        timestamp: 1660346961024
+        version: 5.2.6
+        fn: xz-5.2.6-h57fd34a_0.tar.bz2
+        url: https://conda.anaconda.org/conda-forge/osx-arm64/xz-5.2.6-h57fd34a_0.tar.bz2
+        channel: https://conda.anaconda.org/conda-forge/
+      - build: hc8870d7_0
+        build_number: 0
+        depends: []
+        license: MIT
+        license_family: MIT
+        md5: 8521bd47c0e11c5902535bb1a17c565f
+        name: icu
+        sha256: ff9cd0c6cd1349954c801fb443c94192b637e1b414514539f3c49c56a39f51b1
+        size: 11997841
+        subdir: osx-arm64
+        timestamp: 1692902104771
+        version: "73.2"
+        fn: icu-73.2-hc8870d7_0.conda
+        url: https://conda.anaconda.org/conda-forge/osx-arm64/icu-73.2-hc8870d7_0.conda
+        channel: https://conda.anaconda.org/conda-forge/
+      - build: h44b9a77_0
+        build_number: 0
+        depends:
+          - openssl >=3.0.0,<4.0a0
+        license: MIT
+        license_family: MIT
+        md5: 4a2cac04f86a4540b8c9b8d8f597848f
+        name: sigtool
+        sha256: 70791ae00a3756830cb50451db55f63e2a42a2fa2a8f1bab1ebd36bbb7d55bff
+        size: 210264
+        subdir: osx-arm64
+        timestamp: 1643442231687
+        version: 0.1.3
+        fn: sigtool-0.1.3-h44b9a77_0.tar.bz2
+        url: https://conda.anaconda.org/conda-forge/osx-arm64/sigtool-0.1.3-h44b9a77_0.tar.bz2
+        channel: https://conda.anaconda.org/conda-forge/
+      - build: h0d3ecfb_1
+        build_number: 1
+        constrains:
+          - pyopenssl >=22.1
+        depends:
+          - ca-certificates
+        license: Apache-2.0
+        license_family: Apache
+        md5: eb580fb888d93d5d550c557323ac5cee
+        name: openssl
+        sha256: 519dc941d7ab0ebf31a2878d85c2f444450e7c5f6f41c4d07252c6bb3417b78b
+        size: 2855250
+        subdir: osx-arm64
+        timestamp: 1710793435903
+        version: 3.2.1
+        fn: openssl-3.2.1-h0d3ecfb_1.conda
+        url: https://conda.anaconda.org/conda-forge/osx-arm64/openssl-3.2.1-h0d3ecfb_1.conda
+        channel: https://conda.anaconda.org/conda-forge/
+      - build: hf0a4a13_0
+        build_number: 0
+        depends: []
+        license: ISC
+        md5: fb416a1795f18dcc5a038bc2dc54edf9
+        name: ca-certificates
+        sha256: 49bc3439816ac72d0c0e0f144b8cc870fdcc4adec2e861407ec818d8116b2204
+        size: 155725
+        subdir: osx-arm64
+        timestamp: 1706844034242
+        version: 2024.2.2
+        fn: ca-certificates-2024.2.2-hf0a4a13_0.conda
+        url: https://conda.anaconda.org/conda-forge/osx-arm64/ca-certificates-2024.2.2-hf0a4a13_0.conda
+        channel: https://conda.anaconda.org/conda-forge/
+    run_exports:
```

Comment:
@baszalmstra - we removed these here, right?

Reply:
Yep updated this in 00ad601